src/modules/core/components/MembersList/SortingRow/SortingRow.tsx

```diff
@@ -19,22 +28,51 @@ const MSG = defineMessages({
   },
 });
 
-const SortingRow = () => {
+const SortingRow = ({ handleSortingMethodChange, sortingMethod }: Props) => {
+  const nextSortingByRepMethod =
```

Comment:
`next` because the variable stores the _next_ sorting method that should come after the user clicks on the sorting button. In this case, this const tells us the next sorting method of the reputation sort.